src/Oracles/Config/Setting.hs

```diff
@@ -173,3 +194,26 @@ cmdLineLengthLimit = do
         (False, True) -> 200000
         -- On all other systems, we try this:
         _             -> 4194304 -- Cabal library needs a bit more than 2MB!
+
+-- | https://github.com/ghc/ghc/blob/master/mk/config.mk.in#L241
+relocatableBuild :: Action Bool
+relocatableBuild = windowsHost
+
+prefix, libdir, bindir, datarootdir :: Action String
```

Comment:
By the way, do we really need separate `Action`s for this? I think we could just use `setting InstallPrefix` when needed. I doubt it's used often enough to justify an extra abstraction layer.